src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(double radius) {
+        super(new ColorSupplier().getRandomColor());
```

Comment:
we don't want to always create circles (and other figures) with random color (we can pass it through constructor)

Reply:
How can i make it?

Reply:
`FigureSupplier` can create random color and radious and then pass it to this constructor